neo/rawio/plexonrawio.py

```diff
@@ -264,22 +267,49 @@ def _parse_header(self):
             signal_streams = np.array([], dtype=_signal_stream_dtype)
 
         else:
-            # detect groups (aka streams)
-            all_sig_length = all_sig_length = np.array(all_sig_length)
-            groups = set(zip(sig_channels["sampling_rate"], all_sig_length))
+            # Detect streams
+            all_sig_length = np.asarray(all_sig_length)
+
+            # names are WB{number}, FPX{number}, SPKCX{number}, AI{number}, etc
+            pattern = r"^\D+"  # Match any non-digit character at the beginning of the string
```

Comment:
Thanks for this. I always find regex so hard to read :P The comment is super helpful.

Reply:
Yes. FYI there is this pattern of writing regex expressions like this:

```python
I mean sources for the suggestio to annotate/comment on regex like this:

email_regex = re.compile(r"""
    ^                     # Start of the string
    [a-zA-Z0-9_.+-]+      # Username: allows letters, digits, underscores, dots, pluses, and hyphens
    @                     # The "@" symbol
    [a-zA-Z0-9-]+         # Domain name: allows letters, digits, and hyphens
    (\.[a-zA-Z]{2,})+     # Top-level domain: allows dots followed by 2 or more letters
    $                     # End of the string
""", re.VERBOSE)
```
See the documentation in the docs:
https://docs.python.org/3/library/re.html#re.VERBOSE

Which I advocate for but some people dislike it : P

Reply:
It's helpful, but I get people's point. :)

Reply:
I don't. I think that many programmers fancy themselves as mathematicians and not as the boring manual writters that we are : P